### PR TITLE
Add a name to Line and use it with Navitia providers

### PR DESCRIPTION
**Pull request by grote**
Also add network to Line in Navitia providers

A URL for a line does not seem to be exposed via the Navitia API.

Due to a bug uncovered by adding the network to the line,
the code to retrieve departures had to be simplified.
It now does not do an extra network request to get the line destination.

### CODE REVIEW

**Comment by schildbach** [comment]
Should the name added to the toString()?

File: enabler/src/de/schildbach/pte/dto/Line.java [hunk 1]
```diff
@@ -54,31 +55,58 @@
 
 	public Line(final String id, final String network, final Product product, final String label)
 	{
-		this(id, network, product, label, null, null, null);
+		this(id, network, product, label, null, null, null, null);
+	}
+
+	public Line(final String id, final String network, final Product product, final String label, final String name)
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/dto/Line.java]: I think we only need to add the 1 constructor you're using, and the new version with all parameters. Others can be added as needed.

**Comment by grote** [comment]
I updated the PR. Unfortunately, it now got bigger than originally intended. Let me know what you think.

**Comment by grote** [comment]
Please let me know once you had a chance to look at this one.

**Comment by grote** [comment]
This branch now has conflicts. Want me to rebase?

**Comment by schildbach** [comment]
Yes, please. I just discovered the conflict, too (-:

**Comment by grote** [comment]
Alright, rebased. Was more difficult than expected, so you might want to review carefully.

**Comment by schildbach** [comment]
Thanks, merged.

Just a minor note: The textcolor change could have gone on a separate commit (if not PR), since its entirely independent of the "add name" change. The smaller/more concise a PR is, the easier it is to review and merge.

**Pull request closed without merging**